crates/node-resource-metrics/src/collectors/disk_metrics_collector.rs

```diff
@@ -21,6 +23,19 @@ const NAME_LABEL: &str = "name";
 const TYPE_LABEL: &str = "type";
 const FILE_SYSTEM_LABEL: &str = "file_system";
 
+const LINUX_DISK_SUBSYSTEM: &str = "linux_disk";
+
+const LINUX_NUM_READS: &str = "num_reads";
```

Comment:
It would be good to add some documentation around description of these counters or relevant links here.

Reply:
Sure, I will update the descriptions for the metrics below, so they will also be exported along with the metrics.